modules/codegen/src/main/scala/com/twilio/swagger/codegen/ClientGenerator.scala

```diff
@@ -23,7 +23,7 @@ object ClientGenerator {
       clientExtraImports <- getExtraImports(context.tracing)
       clients <- groupedRoutes.map({ case (pkg, routes) =>
         for {
-          clientCalls <- routes.map(generateClientOperation(pkg, context.tracing, protocolElems) _).sequenceU
+          clientCalls <- routes.map(generateClientOperation(pkg, context.tracing, protocolElems) _).sequence
```

Comment:
Just FYI, `.map(f).sequence` can be written as `.traverse(f)`. It looks like you have several such cases here.